kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/RuntimeErrorHandler.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2010-2022. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.extensions.kafka.eventhandling.consumer;
+
+/**
+ * Used to implement an error handler such that exception from one thread can be passed to another thread.
```

Comment:
```suggestion
 * Used to implement an error handler such that exceptions from one thread can be passed to another thread.
 *
 * @author Gerard Klijs
 * @since 4.5.4
```